src/query/src/promql/planner.rs

```diff
@@ -3112,29 +3163,45 @@ struct FunctionArgs {
     literals: Vec<DfExpr>,
 }
 
+/// Represents different types of scalar functions supported in PromQL expressions.
+/// Each variant defines how the function should be processed and what arguments it expects.
 #[derive(Debug, Clone)]
 enum ScalarFunc {
+    /// DataFusion's built-in scalar functions (e.g., abs, sqrt, round, clamp).
```

Comment:
`clamp` is not built-in, it's more closer to the description of `DataFusionUdf` where "functions registered in DataFusion's function registry"